features/install-profiles/src/main/feature/feature.xml

```diff
@@ -44,13 +44,4 @@
         <feature>spatial-app</feature>
     </feature>
 
-    <feature name="profile-development" version="${project.version}"
```

Comment:
is the development profile no longer needed?

Reply:
The development profile has done the same thing as the standard profile for the past 3 years.  I figured it was time to remove it since no one has fixed it in 3 years. We can always re-add it once someone needs it and has time to fix the issue mentioned. Though I would rather us create something like a docker compose development configuration that could startup LDAP for development testing instead.

https://github.com/codice/ddf/blame/97b7ba9c3278811230be3ad88c7d68c3739b00b8/features/install-profiles/src/main/feature/feature.xml#L52-L53